mono/metadata/object.c

```diff
@@ -3726,13 +3727,19 @@ mono_get_constant_value_from_blob (MonoDomain* domain, MonoTypeEnum type, const
 {
 	MONO_REQ_GC_UNSAFE_MODE
 
+	HANDLE_FUNCTION_ENTER ();
+
+	int result = -1;
+
 	if (!mono_metadata_read_constant_value (blob, type, value, error))
-		return -1;
+		goto exit;
 
 	if (type == MONO_TYPE_STRING)
-		*(gpointer*)value = mono_ldstr_metadata_sig (domain, *(const char**)value, error);
+		*(gpointer*)value = MONO_HANDLE_RAW (mono_ldstr_metadata_sig (domain, *(const char**)value, error));
```

Comment:
This needs a FIXME (or perhaps hold off on changing `mono_get_constant_value_from_blob` until all of its callers are ready to change).

Reply:
Imho FIXMEcoop is best for now.